css/style.css

```diff
@@ -78,6 +78,11 @@ body {
   color: black;
 }
 
```

Comment:
- For the fourth project in your portfolio website, you will: ([Here for the requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md))

>  Use media queries to make sure that your website displays correctly on desktop screens.

- Kindly follow the requirements for this Milestone: 
@media screen and (min-width: 1080px) {
``
your code here
``
}
 **please note that it doesn’t have to be exactly 1080px, any value between 992 and 1100 is ok.**

Reply:
Hello 
Sorry, I made a mistake. 
I have submitted the wrong pull request.